app/models/support/requests/content_change_request.rb

```diff
@@ -22,6 +22,50 @@ def self.label
       def self.description
         "Request changes to GOV.UK content managed by GDS content designers"
       end
+
+      def self.reason_for_change_options
+        [
+          "Factual inaccuracy",
```

Comment:
Is there any reason why these are still hardcoded here instead of being read from the new YAML file?

Reply:
(Sorry, meant to start a review rather than posting a single comment! 🤦‍♂️)

Reply:
That's a great idea, thanks! ⭐  
It means I can delete the config test that prevents them going out of sync.